docs/tutorial.rst

```diff
@@ -1256,7 +1256,7 @@ as aliases, by wrapping them in ``@()``.  For example:
 
     >>> @(_banana)
     'My spoon is tooo big!'
-    >>> echo "hello" | @(lambda args, stdin=None: stdin.strip() + args[0]) world
+    >>> echo "hello" | @(lambda args, stdin=None: ' '.join([stdin.read().strip(), *args]) + '\n') world
```

Comment:
This lambda is more generic (will happily append multiple arguments if given) but lacks the elegance and simplicity of the previous revision :slightly_frowning_face: